client-app/shared/cart/composables/useCheckout.ts

```diff
@@ -0,0 +1,151 @@
+import { Ref, ref, reactive, computed } from "vue";
+import {
+  addOrUpdateCartShipment,
+  addOrUpdateCartPayment,
+  createOrderFromCart,
+  getAvailPaymentMethods,
+  getAvailShippingMethods,
+  removeCart,
+} from "@core/api/graphql/cart";
+import {
+  InputShipmentType,
+  InputPaymentType,
+  InputAddressType,
+  ShippingMethodType,
+  PaymentMethodType,
+  CartAddressType,
+  ShipmentType,
+  PaymentType,
+  CartType,
+} from "@core/api/graphql/types";
+import { Logger } from "@core/utilities";
+import { useCart } from ".";
+
+const addresses: CartAddressType[] = [
+  {
+    id: "1",
+    email: "john@gmail.com",
+    firstName: "John",
+    lastName: "Doe",
+    line1: "Warsawska",
+    line2: "24/193A",
+    city: "Phoenix",
+    postalCode: "26-620",
+    zip: "26-620",
+    countryCode: "US",
+    countryName: "USA",
+    phone: "560123456",
+  },
+];
+
+const loading: Ref<boolean> = ref(false);
+
+const deliveryAddress: Ref<InputAddressType> = ref({ postalCode: "", countryCode: "" });
+const billingAddress: Ref<InputAddressType> = ref({ postalCode: "" });
+
+const chosenShippingMethod: Ref<ShippingMethodType> = ref({});
+const chosenPaymentMethod: Ref<PaymentMethodType> = ref({});
+const shippingAddresses: Ref<CartAddressType[]> = ref(addresses);
+const billingAddresses: Ref<CartAddressType[]> = ref(addresses);
+const shippingMethods: Ref<ShippingMethodType[]> = ref([]);
+
+const paymentMethods: Ref<PaymentMethodType[]> = ref([]);
+const existShipment: Ref<ShipmentType | null> = ref(null);
+const existPayment: Ref<PaymentType | null> = ref(null);
+
+export default () => {
+  async function placeOrder(cartId: string) {
+    const order = await createOrderFromCart(cartId);
+    if (order) {
+      await removeCart(cartId);
+    }
+    return order;
+  }
+
+  async function loadPaymentMethods() {
+    paymentMethods.value = await getAvailPaymentMethods();
+
+    //TODO: remove later
```

Comment:
It's should be removed?

Reply:
@trueboroda  I just moved useCheckout from _reference folder. Didn't touch anything